hack/make.sh

```diff
@@ -327,6 +327,15 @@ copy_containerd() {
 	fi
 }
 
+install_binary() {
+	file="$1"
+	if [ "$(go env GOOS)/$(go env GOARCH)" == "$(go env GOHOSTOS)/$(go env GOHOSTARCH)" ] && [ ! -z "${DOCKER_MAKE_INSTALL}" ]; then
+		echo "Installing $(basename $file)"
+		cp $file /usr/local/bin/
```

Comment:
How about changing `/usr/local/bin` to `${DOCKER_MAKE_INSTALL_PREFIX}/bin` ,and let `${DOCKER_MAKE_INSTALL_PREFIX}` have `/usr/local` as the default value?